Locale/frFR/frFR.lua

```diff
@@ -100,7 +100,7 @@ KiwiItemInfo.Locale["frFR"] = {
 	["KIWIII_RELOAD"] = "recharger",
 	["KIWIII_RESET"] = "réinitialiser",
 	["KIWIII_VARS"] = "vars",
-	["KIWIII_SET"] = "set",
+	["KIWIII_SET"] = "affecte",
```

Comment:
You changed the command. Please reflect the changes in the respective COMMAND_HELP* as well as the usage section in COMMAND_HELP1-3

Reply:
Should I keep these in English? It might make more sense as they are fixed command arguments

Reply:
Or if I translate them, should I keep accented characters? It's very unconventional to have arguments translated in French

Reply:
Whatever you think is best. Spanish people might not know help means ayuda, for example